pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -259,7 +293,7 @@ func syncVolume(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCl
 	return nil
 }
 
-func syncClaim(volumeIndex *persistentVolumeOrderedIndex, binderClient binderClient, claim *api.PersistentVolumeClaim) (err error) {
+func syncClaim(volumeIndex *persistentVolumeOrderedIndex, binderClient binderClient, claim *api.PersistentVolumeClaim, provisionedVolumes map[string]string) (err error) {
```

Comment:
The big comments are in the func itself.

Might be better factored as you suggest.  That would be a separate PR, though, as this is older code with lots of tests, etc. not related to the provisioning feature.